tests/itest_specs/basic_round_trip/short_put.cairo

```diff
@@ -538,22 +552,22 @@ namespace ShortPutRoundTrip {
             contract_address=amm_addr,
             lptoken_address=lpt_call_addr
         );
-        assert call_pool_unlocked_capital_3 = 11529215046068469760;
+        assert call_pool_unlocked_capital_3.low = 5000000000000000000;
 
         // previous unlocked capital was 6734177767761424787928 and the close of half of the option
         // just adds the premia... 6734177767761424787928 + 27.809934 * 2**61
         let (put_pool_unlocked_capital_3) = ILiquidityPool.get_unlocked_capital(
             contract_address=amm_addr,
             lptoken_address=lpt_put_addr
         );
-        assert put_pool_unlocked_capital_3 = 6798303108985114222383;
+        assert put_pool_unlocked_capital_3.low = 2948294000;
 
         // Test balance of option tokens in the account after the option was bought and after withdraw
         let (bal_opt_short_put_tokens_3: Uint256) = IOptionToken.balanceOf(
             contract_address=opt_short_put_addr,
             account=admin_addr
         );
-        assert bal_opt_short_put_tokens_3.low = 500000000000000000;
+        assert bal_opt_short_put_tokens_3.low = 783159565502899114;
```

Comment:
potential BUG!!!

half of the position was closed... by the 
```
        let (premia: Math64x61_) = IAMM.trade_close(
            contract_address=amm_addr,
            option_type=1,
            strike_price=strike_price,
            maturity=expiry,
            option_side=1,
            option_size=half,
            quote_token_address=myusd_addr,
            base_token_address=myeth_addr
        );
```
which means only half of the option tokens should be remaining... how is it possible it is not the case?